omnigan/trainer.py

```diff
@@ -81,16 +81,14 @@ def __init__(self, opts, comet_exp=None, verbose=0, device=None):
 
         self.opts = opts
         self.verbose = verbose
-        self.logger = Dict()
-        self.logger.epoch = 0
+        self.logger = Logger(self)
         self.loaders = None
         self.losses = None
         self.input_shape = None
         self.G = self.D = self.C = None
         self.lr_names = {}
-        self.no_z = self.opts.gen.p.no_z
         self.real_val_fid_stats = None
-
+        self.use_pm4l = False
```

Comment:
Think there's a typo here and you meant "pl4m"?

Reply:
indeed, fixed

Reply:
Indeed